src/XR/webXRManagedOutputCanvas.ts

```diff
@@ -110,11 +110,31 @@ export class WebXRManagedOutputCanvas implements WebXRRenderTarget {
      * @param xrSession xr session
      * @returns a promise that will resolve once the XR Layer has been created
      */
-    public initializeXRLayerAsync(xrSession: XRSession): Promise<XRWebGLLayer> {
+    public initializeXRLayerAsync(xrSession: XRSession): Promise<WebXRLayerWrapper> {

```

Comment:
If we're ok with this breaking change I think it's clean, but I'm also considering not making this breaking change and instead adding some extra functions and using the wrapper type internally only.

Reply:
I think this change is slowly growing on me :-)
But in case it's possible to keep back-compat without major issues, what would be the return type of this function?

Is there a clean way without introducing breaking changes? A unified way of working with both?